ui/pages/confirmation/confirmation.js

```diff
@@ -327,25 +334,46 @@ export default function ConfirmationPage({
             />
           </Box>
         ) : null}
-        {pendingConfirmation.origin === 'metamask' ? null : (
-          <Box
-            alignItems="center"
-            marginTop={1}
-            paddingTop={1}
-            paddingRight={4}
-            paddingLeft={4}
-            paddingBottom={4}
-            flexDirection={FLEX_DIRECTION.COLUMN}
-          >
-            <SiteOrigin
-              chip
-              siteOrigin={originMetadata.origin}
-              title={originMetadata.origin}
-              iconSrc={originMetadata.iconUrl}
-              iconName={originMetadata.hostname}
-            />
-          </Box>
-        )}
+        {
+          ///: BEGIN:ONLY_INCLUDE_IN(flask)
+          !isSnapDialog &&
+            ///: END:ONLY_INCLUDE_IN
+            pendingConfirmation.origin === 'metamask' && (
+              <Box
+                alignItems={AlignItems.center}
+                marginTop={1}
+                paddingTop={1}
+                paddingRight={4}
+                paddingLeft={4}
+                paddingBottom={4}
```

Comment:
same as above, merge please

Reply:
We cannot merge here, one of them is set to `1`